archive/display/src/main/scala/uk/ac/wellcome/platform/archive/display/DisplayIngest.scala

```diff
@@ -44,6 +44,11 @@ case class ResponseDisplayIngest(@JsonKey("@context")
                                  ontologyType: String = "Ingest")
     extends DisplayIngest
 
+case class DisplayIngestMinimal(id: UUID,
```

Comment:
Should this be extending `DisplayIngest`? Or should we be making the fields on `ResponseDisplayIngest` optional?

Reply:
Not sure about this, it can extend `DisplayIngest`, I don't feel this should make fields on `ResponseDisplayIngest` optional.  It could be part of a hierarchy with required fields, but the inclusion of `createdDate` is really just for this use case so I don't feel that's right either.

Reply:
The other option is to extract the core fields into a trait and implement `ResponseDisplayCore` that both case classes implement.